frontend/frontend-sdk/src/lib/client/HttpClient.ts

```diff
@@ -129,13 +130,13 @@ class HttpClient {
   cookie: Cookie;
 
   // eslint-disable-next-line require-jsdoc
-  constructor(api: string, timeout = 13000) {
+  constructor(api: string, options: Options) {
```

Comment:
Same here. There are some more classes, where the `timeout` option is documented and has not been updated to the new version.